lib/utils/index.js

```diff
@@ -727,6 +735,24 @@ function patchFetchFormat(options = {}) {
   }
 }
 
+function build_distribution_domain(options) {
```

Comment:
you can reuse this function in `cloudinary_url()` function, no point to have 2 duplicates of code.